src/scss/components/card/faux-cards/_faux-card--layout--media-text-area.scss

```
@@ -7,26 +7,24 @@
       .paragraph-item.ptype-stanford-wysiwyg.flex-6-of-12 ~ .paragraph-item.ptype-stanford-media-caption.flex-6-of-12,
       // set margin and spacing for media caption on the left/first
       .paragraph-item.ptype-stanford-media-caption.flex-6-of-12 ~ .paragraph-item.ptype-stanford-wysiwyg.flex-6-of-12 {
-        @include grid-media-max(md) {
+        @include grid-media-max('md') {
           margin-top: -$mobile-modular-spacing-2; // negative margin required in this one case, to counteract the previous element's bottom margin, position doesn't work as ewll
           margin-bottom: $mobile-modular-spacing-7;
         }
-        @include grid-media-min(md) {
-          @include grid-media-max(lg) {
-            margin-top: -90px; // to counteract the previous element's bottom margin,
-            @include modular-spacing('margin-bottom', 5);
-          }
+        @include grid-media-between('md', 'lg') {
+          margin-top: -90px; // to counteract the previous element's bottom margin,
+          @include modular-spacing('margin-bottom', 5);
```

```suggestion
          @include modular-spacing('margin', null null 5);
```
That's the way we usually write it, but glad to know that your way works too 😄

---

Yes, I was doing it that way too, then saw this other way which is a little more concise. If we're just setting one i think i'll do the specific way, otherwise do the full way.